listitemview/src/main/AndroidManifest.xml

```diff
@@ -1,11 +1,6 @@
-<manifest package="com.lucasurbas.listitemview"
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.lucasurbas.listitemview">
 
-    xmlns:android="http://schemas.android.com/apk/res/android">
-
-    <application android:allowBackup="true"
-        android:label="@string/app_name"
-        android:supportsRtl="true">
-
-    </application>
+    <application android:supportsRtl="true" />
```

Comment:
also RTL should not be forced. This lib supports RTL but this does not mean all apps using this lib support RTL

Reply:
You are right!